src/libraries/System.Console/tests/ManualTests/ManualTests.cs

```diff
@@ -44,6 +44,21 @@ public static void ReadLineFromOpenStandardInput()
             AssertUserExpectedResults("the characters you typed properly echoed as you typed");
         }
 
+        [ConditionalFact(nameof(ManualTestsEnabled))]
+        public static void ConsoleReadSupportsBackspace()
```

Comment:
@eiriktsarpalis can you please run this test on Windows and see if it passes?

Reply:
It fails on my box, with the following message:
```
Please type 'a' 3 times, press 'Backspace' to erase 1, then type a single 'b' and press 'Enter'.
aab
Please press 'a' until it wraps to the next terminal line, then press 'Backspace' until the input is erased, and then type a single 'a' and press 'Enter'.
Input: Use the up, down, left, and right arrow keys to move around.  When done, press enter.
    System.ConsoleManualTests.ConsoleReadSupportsBackspace [FAIL]
      Assert.Equal() Failure
      Expected: 10
      Actual:   13
      Stack Trace:
        C:\Users\eitsarpa\devel\dotnet\runtime\src\libraries\System.Console\tests\ManualTests\ManualTests.cs(56,0): at System.ConsoleManualTests.ConsoleReadSupportsBackspace()
    System.ConsoleManualTests.ReadLine_BackSpaceCanMoveAcrossWrappedLines [FAIL]
      Assert.Equal() Failure
                ↓ (pos 0)
      Expected: a
      Actual:
                ↑ (pos 0)
      Stack Trace:
        C:\Users\eitsarpa\devel\dotnet\runtime\src\libraries\System.Console\tests\ManualTests\ManualTests.cs(70,0): at System.ConsoleManualTests.ReadLine_BackSpaceCanMoveAcrossWrappedLines()
```

Reply:
FYI I'm no longer owner of System.Console, cc the current owners @adamsitnik @Jozkee and @carlossanlop.

Reply:
> It fails on my box, with the following message:

I pushed a commit that should fix this.

> FYI I'm no longer owner of System.Console, cc the current owners @adamsitnik @Jozkee and @carlossanlop.

ok, thanks for running the test!

Current owners ptal.